src/Components/CreateImageWizard/ValidatedTextInput.tsx

```diff
@@ -19,16 +21,18 @@ interface ValidatedTextInputPropTypes extends TextInputProps {
   placeholder?: string;
 }
 
-interface HookValidatedTextInputPropTypes extends TextInputProps {
-  dataTestId?: string | undefined;
-  ouiaId?: string;
-  ariaLabel: string | undefined;
-  value: string;
-  placeholder?: string;
-  stepValidation: StepValidation;
-  fieldName: string;
-  warning?: string;
-}
+type HookValidatedTextInputPropTypes = TextInputProps &
```

Comment:
We probably need to rename this component now that it is no longer a text input but supports both `input` and `area`. Maybe just `HookValidatedInput`.